pipelines/HyperSuprimeCam/ApTemplate.yaml

```diff
@@ -17,6 +17,19 @@ tasks:
       doApplyExternalPhotoCalib: False
       doApplyExternalSkyWcs: False
       makePsfMatched: True
+  assembleCoadd:
+    class: lsst.pipe.tasks.assembleCoadd.CompareWarpAssembleCoaddTask
+    config:
+      file: $AP_PIPE_DIR/config/HSC/assembleCoadd.py
+      # Copy base pipeline's configs, so that they override HSC settings.
```

Comment:
This comment worries me: is the problem that the `ApTemplate.yaml` configs get overridden by obs_subaru which then gets overridden by `config/HSC/assembleCoadd.py` so you have to reset them here? Is this something that we'll just fix directly when we start configuring this all for ourselves and not have so many layers of indirection?

Reply:
Yes. In particular, the "ignore obs packages" hack I added later _also_ wipes out the configs from the base pipeline. I don't see a way around this until the obs configs either go away or stop being auto-loaded by the execution framework.

Reply:
Thanks: I think the comment could be clarified a bit to make it more clear exactly what is being overridden, since it wasn't obvious to me as I was going through this.